crates/meroctl/src/cli/context/common.rs

```diff
@@ -31,8 +31,7 @@ pub(crate) fn multiaddr_to_url(
 
     let mut url = Url::parse(&format!("{}://{}:{}", scheme, ip, port))?;
 
-    if let Some(api_path) = api_path {
-        url.set_path(&api_path);
-    }
+    url.set_path(&api_path);
```

Comment:
looks like api_path doesn't need an owned string either, should be fine as a `&str`